### PR TITLE
Labelling guide

### PR DESCRIPTION
**Pull request by m-allanson**
A guide to labelling issues on Gatsby's GitHub repo. WIP - comments welcome!

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit 14e99bae11daed60592c11e2fbf90e67e86d6f01

https://app.netlify.com/sites/using-postcss-sass/deploys/5b59ba16c9659269f32dc288

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit 14e99bae11daed60592c11e2fbf90e67e86d6f01

https://app.netlify.com/sites/gatsbyjs/deploys/5b59ba15c9659269f32dc280

File: docs/docs/labelling-issues.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+- reported and confirmed bugs
+- new features
+- duplicate issues
+- issues that are stalled or blocked
+
+## Who can label issues?
+
+Anyone who's a member of the [Gatsby Maintainers team](https://github.com/orgs/gatsbyjs/teams/maintainers) can label issues.
+
+You can get an invite to the team by having a Pull Request merged into the Gatsby project. Check out the list of [`help wanted`](https://github.com/gatsbyjs/gatsby/labels/%F0%9F%93%8D%20status%3A%20help%20wanted) issues and the [How To Contribute Guide](/docs/how-to-contribute/) to get started.
```
Comment (jlengstorf) [inline docs/docs/labelling-issues.md]: Let's add a note:

**NOTE:** If you’ve already had a pull request merged and you have _not_ been invited to the maintainers team, please go to [the dashboard](https://store.gatsbyjs.org/account/dashboard) and request a discount code. You should get an invite to the team — _and you get free Gatsby swag!_ If that doesn’t work, please email team@gatsbyjs.com and we'll get you invited.

File: docs/docs/labelling-issues.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+Ideally every issue should have a single `🏷 type:` label applied to it. Optionally a `status: ` label, or other labels may also be applied.
+
+Before continuing, get familiar with [Gatsby's issue labels and their descriptions](https://github.com/gatsbyjs/gatsby/issues/labels).
+
+The broad steps to labelling an issue are:
+
+- Read an issue
+- Choose the labels that apply to that issue
+- That's it
```
Comment (jlengstorf) [inline docs/docs/labelling-issues.md]: Can we add an exclamation point or emoji here to make it a bit more playful?

File: docs/docs/labelling-issues.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+### Choose one `type: ` label
+
+Choose a type label from the _labels_ dropdown to the right-hand side of the issue.
+
+!(GitHub label dropdown)(./images/github-label-list.png)
+
+You can check through the [label descriptions](https://github.com/gatsbyjs/gatsby/issues/labels) for more information on each one.
+
+The most common type of issue is `type: question or discussion`, typically you can apply this to issues that are open-ended or have no clear next step.
```
Comment (jlengstorf) [inline docs/docs/labelling-issues.md]: This is kind of tangential, but are these the issues we eventually want to move _off_ the issue queue? My feeling is that if an issue won't result in changes to the Gatsby core, it's probably better for a support channel, not GitHub issues.

File: docs/docs/labelling-issues.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+- An issue with a clear description of how it can be resolved could be labelled `📍 status: help wanted`.
+
+- An issue that's missing information required to help the author could be labelled with `📍 status: needs more info`
+
+- An issue describing a bug without clear steps to reproduce could be labelled with `📍 status: needs reproduction`
+
+- An issue describing a bug where there's steps to reproduce the bug _and_ you (or someone else) have confirmed that they can reproduce the bug can be labelled `📍 status: confirmed`
+
+### TODO - is there a way to publicly list these people?  https://github.com/orgs/gatsbyjs/teams/admin/members
+- An issue describing a feature can be labelled `📍 status: confirmed` if a member of Gatsby's Admin team has approved the feature
```
Comment (jlengstorf) [inline docs/docs/labelling-issues.md]: Not that I've seen, unfortunately. Teams are pretty much invisible unless you're part of them.

File: docs/docs/labelling-issues.md (hunk 2 above)
Comment (m-allanson) [reply docs/docs/labelling-issues.md]: I very nearly went for the classic 💜 💪, but in the [style guide](https://next.gatsbyjs.org/docs/gatsby-style-guide/) it says:

> Don’t use emojis or emoticons in the Guide. Gatsby has a global community, and the cultural meaning of an emoji or emoticon may be different around the world. Also, emojis can render differently on different systems.

Maybe we can break that rule occasionally?

File: docs/docs/labelling-issues.md (hunk 3 above)
Comment (m-allanson) [reply docs/docs/labelling-issues.md]: Agreed, but we don't have a specific support channel set up for this yet.

File: docs/docs/labelling-issues.md (hunk 4 above)
Comment (m-allanson) [reply docs/docs/labelling-issues.md]: Hrmm ok, I'll try and reword that somehow... "Don't do this unless you know you're allowed to" isn't too helpful though! Maybe we can drop this line.

File: docs/docs/labelling-issues.md (hunk 2 above)
Comment (m-allanson) [reply docs/docs/labelling-issues.md]: I attempted to lighten the mood with words instead of emoji here

**Comment by m-allanson** [comment]
I've updated this to address feedback and linked it up in the sidebar

File: docs/docs/how-to-label-an-issue.md [hunk 5]
```diff
@@ -35,7 +37,7 @@ The broad steps to labelling an issue are:
 
 - Read an issue
 - Choose the labels that apply to that issue
-- That's it
+- That's it - sit back and relax, maybe take a few moments to enjoy the satisfaction of a job well done
```
Comment (jlengstorf) [inline docs/docs/how-to-label-an-issue.md]: 💜

File: docs/docs/how-to-label-an-issue.md [hunk 6]
```diff
@@ -25,6 +25,8 @@ Anyone who's a member of the [Gatsby Maintainers team](https://github.com/orgs/g
 
 You can get an invite to the team by having a Pull Request merged into the Gatsby project. Check out the list of [`help wanted`](https://github.com/gatsbyjs/gatsby/labels/%F0%9F%93%8D%20status%3A%20help%20wanted) issues and the [How To Contribute Guide](/docs/how-to-contribute/) to get started.
 
+**NOTE:** If you’ve already had a pull request merged and you have _not_ been invited to the maintainers team, please go to [the dashboard](https://store.gatsbyjs.org/account/dashboard) and request a discount code. You should get an invite to the team — _and you get free Gatsby swag!_ If that doesn’t work, please email team@gatsbyjs.com and we'll get you invited.
+
 ## How to label an issue
 
 Ideally every issue should have a single `🏷 type:` label applied to it. Optionally a `status: ` label, or other labels may also be applied.
```
Comment (jlengstorf) [inline docs/docs/how-to-label-an-issue.md]: Should we remove the emoji here?

Comment (m-allanson) [reply docs/docs/how-to-label-an-issue.md]: Oh yes, good catch thanks!

**Comment by m-allanson** [comment]
Thanks for the reviews everyone 👍 

I don't think this needs a re-review just for deleting an emoji, so I'm going to call this done and merge it.

File: docs/docs/labelling-issues.md (hunk 2 above)
Comment (shannonbux) [reply docs/docs/labelling-issues.md]: Hm, I have no idea...just looked up this article and wondering how much internet culture reduces some misunderstandings (e.g. if someone grew up using emojis w/ people all across the world, there's a chance all will have a shared understanding. However, it's hard to be sure!) https://www.airportparkingreservations.com/blog/avoid-using-these-hand-gestures-and-emojis-in-other-countries/

^^^ Also I'd have to do some more thorough linguistics research bc that article could be wrong.

Looks like 💪💜 these don't mean anything though!

**Pull request merged**